src/main/resources/application.yml

```diff
@@ -17,4 +17,8 @@ management:
     web:
       exposure.include: info, health
 
-apiKey.bizEvents: ${BIZ_EVENTS_SERVICE_API_KEY:}
\ No newline at end of file
+apiKey.biz-events: \${BIZ_EVENTS_SERVICE_API_KEY:}
+
+rest-client:
+  biz-events:
+    baseUrl: \${BIZ_EVENTS_BASE_URL:}
```

Comment:
i think we can group the properties related to biz-events under the same root property